python/cuml/common/opg_data_utils_mg.pyx

```diff
@@ -140,3 +141,15 @@ def free_rank_size_pair(rank_size_t, n):
     for idx in range(n):
         free(<RankSizePair*>rankSizePair[idx])
     free(<RankSizePair**>rankSizePair)
+
+
+def build_pred_or_trans_arys(arys, order, dtype):
```

Comment:
```suggestion
def build_zeros_cumlarrays_like(arys, order, dtype):
```
A more descriptive name would be nice

Reply:
On second thought, the exact function (`cumlarray.zeros`) could be a parameter to this function and then this could allow users to build lists of cumlarrays empty/initialized/etc however they require.

Reply:
Could be a future improvement, since otherwise the PR looks good, please open a github issue :)